packages/contracts/scripts/smt-checker/README.md

```diff
@@ -0,0 +1,62 @@
+# SMT-Checker Support for Smart Contract Testing
+
+This repository provides SMT-Checker support for testing smart contracts. Follow these steps to get started:
+
+
+## Make Sure Your System Has Z3 Installed >= "4.8.11"
+
+Ensure that your system has Z3 installed with a version number greater than or equal to the required version.
+
+## Step 1: Download and Install Z3
+
+Before proceeding, ensure that you have `g++` and `python3` installed on your system. To download and install Z3, enter the following commands in your terminal:
+
+```
+sudo apt-get install g++
+sudo apt-get install z3
+sudo apt-get install libz3-dev
+sudo wget https://github.com/Z3Prover/z3/archive/refs/tags/z3-4.11.0.tar.gz
+sudo tar -zxvf z3-4.11.0.tar.gz
+cd z3-z3-4.11.0
+python3 scripts/mk_make.py
+cd build
+sudo make -j$(nproc)
+sudo make install
+./smt-checker/ubiquity-dollar/packages/contracts/smt-checker/copy.sh
+```
```

Comment:
All this can be part of a shell script instead

Reply:
https://github.com/ubiquity/ubiquity-dollar/commit/aa2109926197aed781042491de602f47532b6db3